packages/scripts/utils/cli.js

```diff
@@ -24,6 +24,12 @@ const getCliArg = ( arg ) => {
 	}
 };
 
+const cleanUpArgs = ( prefixes ) => {
```

Comment:
This function gets the list of arguments and filters it. The name doesn't reflect it. How about we name it `getFilteredCliArgs`? Another option is to bake in this filtering into the original `getCliArgs`.

```js
function getCliArgs( { exclude } ) {
    ....
   if ( exclude ) {
       // remove listed args
   }
   return filteredArgs;
}   
```

Reply:
I think adding exclude into getCliArgs is the best way indeed. I'll push some changes to reflect that!